budget_invoice/model/budget_line.py

```diff
@@ -46,10 +46,34 @@ def invoice_create(self, cr, uid, ids, product, partner, context=None):
         invoice_id = invoice_obj.create(cr, uid, invoice_data, context)
 
         for budget_line in self.browse(cr, uid, ids, context):
+            product_onchange_result = invoice_line_obj.product_id_change(
```

Comment:
just v7 to v8 migration for modules to be available.
maybe in a next PR specific for v8 api ?